pkg/virt-api/rest/expand_test.go

```diff
@@ -139,11 +144,13 @@ var _ = Describe("Instancetype expansion subresources", func() {
 
 			expectedVm := vm.DeepCopy()
 			expectedVm.Spec.Template.Spec.Domain.CPU = cpu
+			expectedVm.Spec.Template.ObjectMeta.Annotations = annotations
 
 			recorder := callExpandSpecApi(vm)
 			responseVm := &v1.VirtualMachine{}
 			Expect(json.NewDecoder(recorder.Body).Decode(responseVm)).To(Succeed())
 
+			Expect(responseVm.Spec.Template.ObjectMeta).To(Equal(expectedVm.Spec.Template.ObjectMeta))
```

Comment:
Limit this to `Spec.Template.ObjectMeta.Annotations`?

Reply:
Done